### DISCUSSION TITLE
Share a working v28 localhost here

### DISCUSSION THREAD

**Discussion by ShanQincheng** [Hucaru/Valhalla#67] [Show and tell]
I spent the whole day debugging an 'UNKNOWN CLIENT PACKET' error and found out that I was using an incompatible v28 localhost client. 

So, I am sharing a working one here that is compatible with the Valhalla project.
[v28_noaes.zip](https://github.com/Hucaru/Valhalla/files/14207305/v28_noaes.zip)

**Comment by xlanor** [comment]
[v28_noaes_patched_res_noie_mp.zip](https://github.com/user-attachments/files/19866472/v28_noaes_patched_res_noie_mp.zip)
Here's a copy with all the client patches applied as per README.
